src/Microsoft.Crank.Agent/Startup.cs

```diff
@@ -1867,6 +1881,61 @@ private static async Task<(string containerId, string imageName, string workingD
                     {
                         job.Error = job.BuildLog.ToString();
                     }
+
+                    var dockerInspectArguments = $"inspect -f \"{{{{ .Size }}}}\" {imageName}";
+
+                    var inspectResults = await ProcessUtil.RunAsync("docker", dockerInspectArguments,
+                        workingDirectory: srcDir,
+                        cancellationToken: cancellationToken,
+                        captureOutput: true,
+                        log: true,
+                        outputDataReceived: text => job.BuildLog.AddLine(text));
+
+                    if(long.TryParse(inspectResults.StandardOutput.Trim(), out var imageSize))
+                    {
+                        if (imageSize != 0)
+                        {
+                            job.PublishedSize = imageSize;
+
+                            job.Measurements.Enqueue(new Measurement
+                            {
+                                Name = "benchmarks/published-size",
+                                Timestamp = DateTime.UtcNow,
+                                Value = imageSize
+                            });
+                        }
+                    }
+
+                    var dockerSaveArguments = $"save {imageName} -o {imageName}.tar";
```

Comment:
Why should we store the compressed size? Tracking it won't be different than tracking the non-compressed one? Any regression in one will be seen in the other. I assume "we" already know what is the size of the dotnet images already. If it was free that would not matter much, but here we zip the 200MB for no value (IMO, until convinced otherwise).

Reply:
I think understanding the impact to compressed size of various aot/linker options is interesing. @mangod9 what do you think?

Reply:
Also @richlander

Reply:
is the concern that we a single metric is too costly to store? Ok with not storing at the moment.

Reply:
I think the concern is the time to collect it. We spend an extra 10 seconds or so to create the zip file.

@sebastienros its gonna be the same file for every run in this mix. What if I make it a conditional based on some property like I originally suggested and just do it for a single test? That way the whole matrix doesn't spend this time.

Reply:
It has a cost of time to serialize the image to a file, then run the command to compress it, for each run of the matrix, just to know what is the size of the container once compressed. I don't see the value of computing it on each run. It should be a known value. Perf changes won't impact the result run after run, and raw publish size changes will show us any regression in size.

Reply:
Still seems odd. It shouldn't be the agent's concern to measure such an artificial value. I am not sure anyone has told me yet what value it provides to have it there.

Please educate me. Even outside of the agent, do we want to know what is the compressed size of the final image with the app? If so do we know it already? Either we don't know in which case that can't be that important and we can do a single run to know, or we do and that won't change much more than the uncompressed size.

An argument I could accept is that we want to compare the same app to a different framework and see how big the image is. But how much value does it add to comparing the images sizes in their uncompressed state. And how much flexibility do we have to improve the compressed size "more" than the uncompressed size?

The code is there, and we can just merge the PR as-is, but my argument is more articulated reasons to measure something, and the value it adds. 

@billwert I am ok with adding an argument in the Job (`bool MeasureCompressedDockerSize`, undocumented), running it for all the scenarios (easier to chart than a single one), and in a few weeks we look at what knowledge all these daily measurements got us, and if there is any reason to keep this running.

Reply:
It would have been handy to have had it for blogging purposes. That said, I don't feel too strongly about it and I've just removed the code.

Can we merge?